### PR TITLE
Best Practices: Updating examples to use "controllerAs" syntax.

### PR DESCRIPTION
**Pull request by johnlindquist**
Igor sent me here :) Once this gets signed off, I'll update the description/annotation text as well then move on to the component/directive example.

### CODE REVIEW

**Comment by caitp** [comment]
ping @johnlindquist did you want to fix up the bitrot and make that tiny adjustment? looks good to me as well

File: src/index.html [hunk 1]
```diff
@@ -472,46 +472,47 @@ <h1 id="embed-and-inject">Embed and Inject</h1>
...
-        }]);
+        });
     </script>
     <script type="text/ng-template" id="todo.html">
       <h2>Todo</h2>
-      <div ng-controller="TodoController">
-        <span>{{remaining()}} of {{todos.length}} remaining</span>
-        [ <a href="" ng-click="archive()">archive</a> ]
+      <div ng-controller="TodoListCtrl as todoList">
+        <span>{{todoList.remaining()}} of {{todos.length}} remaining</span>
```
Comment (gkalpak) [inline src/index.html]: _todos.length_  -->  _todoList.todos.length_

File: src/index.html [hunk 2]
```diff
@@ -586,35 +587,38 @@ <h1 id="embed-and-inject">Embed and Inject</h1>
...
-        $scope.projects = Projects;
+      .controller('ProjectListCtrl', function(Projects) {
+        var projectList = this;
+        projectList.projects = Projects;
       })
 
-      .controller('CreateCtrl', function($scope, $location, $timeout, Projects) {
-        $scope.save = function() {
-            Projects.$add($scope.project).then(function(data) {
+      .controller('NewProjectCtrl', function($location, $timeout, Projects) {
```
Comment (gkalpak) [inline src/index.html]: Is `$timeout` needed ?

File: src/index.html [hunk 3]
```diff
@@ -649,14 +653,14 @@ <h1 id="embed-and-inject">Embed and Inject</h1>
       <form name="myForm">
         <div class="control-group" ng-class="{error: myForm.name.$invalid && !myForm.name.$pristine}">
           <label>Name</label>
-          <input type="text" name="name" ng-model="project.name" required>
+          <input type="text" name="name" ng-model="editProject.project.name" required>
           <span ng-show="myForm.name.$error.required && !myForm.name.$pristine" class="help-inline">
               Required {{myForm.name.$pristine}}</span>
```
Comment (gkalpak) [inline src/index.html]: What purpose is `{{myForm.name.$pristine}}` supposed to serve here ?
Is it a typo ?

**Pull request closed without merging**